doc/JSON_FLAGS.md

```diff
@@ -564,6 +564,7 @@ List of known flags, used in both `terrain.json` and `furniture.json`.
 - ```NOCOLLIDE``` Feature that simply doesn't collide with vehicles at all.
 - ```NOITEM``` Items cannot be added here but may overflow to adjacent tiles. See also `DESTROY_ITEM`
 - ```NO_FLOOR``` Things should fall when placed on this tile
+- ```NO_PICKUP_ON_EXAMINE``` <kbd>e</kbd>xamining this tile won't open Pick Up menu even if there are items here.
```

Comment:
```suggestion
- ```NO_PICKUP_ON_EXAMINE``` examining this tile won't open Pick Up menu even if there are items here.
```

Reply:
There are a few other places (not related to this PR) in the markdown docs with `<kbd>`:

```
Search "<kbd>" (6 hits in 2 files of 67 searched)
  C:\Projects\Cataclysm-DDA\doc\ITEM_SPAWN.md (3 hits)
	Line 247:     > *TIP*: If a key isn't bound to the debug menu or you forgot it, use <kbd>ESC</kbd> > <kbd>1</kbd>.
	Line 247:     > *TIP*: If a key isn't bound to the debug menu or you forgot it, use <kbd>ESC</kbd> > <kbd>1</kbd>.
	Line 255:     > *TIP*: You can filter anything in the debug menu using <kbd>/</kbd>.
  C:\Projects\Cataclysm-DDA\doc\JSON_FLAGS.md (3 hits)
	Line 550: - ```GOES_DOWN``` Can use <kbd>></kbd> to go down a level.
	Line 551: - ```GOES_UP``` Can use <kbd><</kbd> to go up a level.
	Line 586: - ```SEALED``` Can't use <kbd>e</kbd> to retrieve items; must smash them open first.
```

Reply:
Why remove this `kbd`? I thought it's used to mark the letter.

Reply:
I mean keybinding could be different - maybe we better mention command name first.